codeSnippets/snippets/proguard/README.md

```diff
@@ -3,12 +3,30 @@
 A sample project packed as a JAR using the [Ktor Gradle plugin](https://ktor.io/docs/fatjar.html) and minimized using [ProGuard](https://www.guardsquare.com/manual/home).
 > This sample is a part of the [codeSnippets](../../README.md) Gradle project.
 
+This example is configured to work with Kotlinx coroutines and serialization, required modules that are part of the JDK
+and some libraries that Ktor server depends on.
+
+if you're using additional libraries that use features such as 
+reflections or dynamic class loading, You will get a warning that needs to be solved.
```

Comment:
Here do you mean that the user would need to include more rules to fix the warning?

Reply:
> Here do you mean that the user would need to include more rules to fix the warning?

The developer probably use more libraries that are not part of the example, he or she might not familiar with Proguard as it's known as Android tool,including this note to indicate that the warnings as Proguard won't be able to solve any usages that use dynamic access (dynamic class loading) or reflections and need to be solved.

Proguard will be able to detect most common usages, see (https://www.guardsquare.com/manual/home) for more details.

The example is shown in the documentation of the Ktor server, so it's why I included details that's related to Proguard.

Let me know if I should remove this note.

Reply:
Thanks for clarifying. You can keep the note and add a reference to the Proguard docs for more information.